test/dynamo/test_logging.py

```diff
@@ -388,10 +388,10 @@ def outer(pred, x):
             msg = record.getMessage()
             if "return x * 2" in msg:
                 found_x2 = True
-                self.assertIn("inline depth: 2", msg)
+                self.assertIn("inline depth: 3", msg)
```

Comment:
torch._higher_order_ops.cond.cond  is treated as a UserFunctionVariable by dynamo. so it increases the inline depth by one